doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1354 @@
+    * The module implementation to use :c:func:`sys_poweroff` API instead of :c:func:`pm_state_force` API to enter the system off state.
+    * The module implementation to integrate the newly introduced :c:struct:`power_off_event`.
+      The event is used to inform application modules that system power off (:c:func:`sys_poweroff`) is about to happen.
+
+DFU libraries
+-------------
+
+* Added:
+
+  * Support for :ref:`manifest-controlled variables<subsys_suit_mfst_var>` that allow to control manifest logic based on previous evaluations as well as store integer values inside the SUIT non-volatile memory region.
+  * Support for :ref:`In-place Updateable Components (IPUC)<ug_nrf54h20_suit_ipuc>` that allow to cross memory permission boundaries to update inactive memory regions from the main application.
+
+* :ref:`lib_fmfu_fdev`:
+
+  * Regenerated the zcbor-generated code files using v0.9.0.
+
+* :ref:`subsys_suit`:
+
+   * Added:
+
+     * Support for manifest-controlled variables, that allow to control manifest logic based on previous evaluations as well as store integer values inside the SUIT non-volatile memory region.
+     * Support for in-place updateable components (IPUC) that allows to cross memory permission boundaries to update inactive memory regions from the main application.
+     * Support for IPUC in SUIT manifests that makes possible the following:
+
+       * To fetch payloads directly into an IPUC.
+       * To declare an IPUC as DFU cache area.
+       * To trigger Nordic firmware updates from an IPUC-based DFU cache area.
+
+     * Support for IPUC in DFU protocols that makes possible the following:
+
+       * To write into IPUC using SMP image command group.
+       * To write into IPUC using SUIT SMP cache raw upload commands.
+       * To write into IPUC using SUIT :ref:`dfu_target<lib_dfu_target_suit_style_update>` library.
+
+     * Possibilities for the following:
+
+       * To copy binaries into radio local RAM memory from SUIT radio manifests.
+       * To specify the minimal Nordic top manifest version in the :file:`VERSION` file.
+       * To block independent updates of Nordic manifests using the :kconfig:option:`CONFIG_SUIT_NORDIC_TOP_INDEPENDENT_UPDATE_FORBIDDEN` Kconfig option.
+
+   * Updated by moving the MPI configuration from local Kconfig options to sysbuild.
+
+Modem libraries
+---------------
+
+* Deprecated the :ref:`at_params_readme` library.
+
+* :ref:`pdn_readme` library:
+
+  * Deprecated the :c:func:`pdn_dynamic_params_get` function.
+    Use the new function :c:func:`pdn_dynamic_info_get` instead.
+
+* :ref:`lte_lc_readme` library:
+
+  * Fixed handling of ``%NCELLMEAS`` notification with status 2 (measurement interrupted) and no cells.
+  * Added sending of ``LTE_LC_EVT_NEIGHBOR_CELL_MEAS`` event with ``current_cell`` set to ``LTE_LC_CELL_EUTRAN_ID_INVALID`` in case an error occurs while parsing the ``%NCELLMEAS`` notification.
```

Comment:
```suggestion
  * Added sending of ``LTE_LC_EVT_NEIGHBOR_CELL_MEAS`` event with ``current_cell`` set to ``LTE_LC_CELL_EUTRAN_ID_INVALID`` in case an error occurs while parsing the ``%NCELLMEAS`` notification.
  * Fixed handling of ``%NCELLMEAS`` notification with status 2 (measurement interrupted) and no cells.
```